src/LowMachEquationSystem.C

```diff
@@ -2480,8 +2480,13 @@ MomentumEquationSystem::register_overset_bc()
 void
 MomentumEquationSystem::initialize()
 {
+  for (auto it = equationSystems_.equationSystemVector_.begin(); it != equationSystems_.equationSystemVector_.end(); ++it) {
+    //std::cout << "MomentumEquationSystem::initialize() : equationSystem name = " <<  (*it)->name_ << "," << (*it)->convergenceTolerance_ << std::endl;
+    if ((*it)->name_ == "LowMachEOSWrap")
+      convergenceTolerance_ = (*it)->convergenceTolerance_;
+  }
```

Comment:
Why is this here? The `convergenceTolerance_` values are set here 

https://github.com/Exawind/nalu-wind/blob/b9d2834c770be4786e23f1d32e28f718af82c991/src/LowMachEquationSystem.C#L320-L326

Reply:
@sayerhs With the refactor, this code is no longer being called, hence momentum and continuity are defaulting to a NL tolerance of 1.

Reply:
Here are the backtraces that show where the flow of control diverges:

Nalu-Wind master

```
#1  0x000000000064972f in sierra::nalu::EquationSystems::initialize (this=this@entry=0x141ec30)
    at /home/jhu/exawind/sources/nalu-wind-exawind/src/EquationSystems.C:713
713     eqSys->initialize();
(gdb)
#2  0x000000000053ab19 in sierra::nalu::Realm::initialize (this=0x141e970) at /home/jhu/exawind/sources/nalu-wind-exawind/src/Realm.C:566
566   equationSystems_.initialize();
(gdb)
#3  0x0000000000548626 in sierra::nalu::Realms::initialize (this=0x144eb00) at /home/jhu/exawind/sources/nalu-wind-exawind/src/Realms.C:78
78      realmVector_[irealm]->initialize();
(gdb)
#4  0x0000000000513d4d in sierra::nalu::Simulation::initialize (this=this@entry=0x7fffffffa750)
    at /home/jhu/exawind/sources/nalu-wind-exawind/src/Simulation.C:146
146   realms_->initialize();
(gdb)
#5  0x000000000045cf01 in main (argc=<optimized out>, argv=<optimized out>) at /home/jhu/exawind/sources/nalu-wind-exawind/nalu.C:168
168   sim.initialize();
```

CrsGraph branch

```
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/LowMachEquationSystem.C:3687
3687	{
(gdb) n
3688	  for (auto it = equationSystems_.equationSystemVector_.begin(); it != equationSystems_.equationSystemVector_.end(); ++it) {
(gdb) up
#1  0x000000000064dd8f in sierra::nalu::EquationSystems::<lambda(sierra::nalu::EquationSystem*)>::operator() (
    __closure=<optimized out>, eqsys=0x16576d0) at /home/jhu/exawind/sources/nalu-wind-jhux2/src/EquationSystems.C:776
776	      eqsys->initialize();
(gdb)
#2  sierra::nalu::(anonymous namespace)::initialize_connectivity<sierra::nalu::EquationSystems::initialize()::<lambda(sierra::nalu::EquationSystem*)> > (eqSysVec=std::vector of length 4, capacity 4 = {...}, realm=..., func=...)
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/EquationSystems.C:97
97	      func(eqsys);
(gdb)
#3  sierra::nalu::EquationSystems::initialize (this=this@entry=0x1421350)
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/EquationSystems.C:772
772	  initialize_connectivity(
(gdb)
#4  0x000000000053c769 in sierra::nalu::Realm::initialize (this=0x1421090)
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/Realm.C:566
566	  equationSystems_.initialize();
(gdb)
#5  0x000000000054a2d6 in sierra::nalu::Realms::initialize (this=0x1451640)
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/Realms.C:78
78	    realmVector_[irealm]->initialize();
(gdb)
#6  0x000000000051593d in sierra::nalu::Simulation::initialize (this=this@entry=0x7fffffffa750)
    at /home/jhu/exawind/sources/nalu-wind-jhux2/src/Simulation.C:146
146	  realms_->initialize();
```